console/src/main/webapp/manager/app/templates/userForm.tpl.html

```diff
@@ -126,18 +126,28 @@
           <input ng-model="model.uid" class="form-control" id="uid" placeholder="{{'user.login' | translate}}" ng-required="required.uid" ng-pattern="/^[a-z][a-z0-9_\\\\.\\\\-]*$/i">
         </div>
       </div>
+      <hr>
 
-      <div class="form-group" ng-if="model.$promise" ng-class="{required: required.preferredLanguage}">
+      <div class="form-group form-group-sm" ng-if="model.$promise" ng-class="{required: required.preferredLanguage}">
         <label class="col-sm-4" for="givenName" translate>user.password</label>
         <a href="/console/account/passwordRecovery?email={{user.user.mail | encodeURIComponent}}"
           target="_blank" class="col-sm-8 password">
           <span translate>user.reset</span>
           <i class="glyphicon glyphicon-new-window"></i>
         </a>
       </div>
-
+      <div ng-if="platformInfos.saslEnabled">
+        <div class="form-group form-group-sm" ng-if="model.$promise">
+          <label class="col-sm-4" for="saslUser">
+            <span translate>sasl.remote.user</span>
+            {{ platformInfos.saslServer }}
+          </label>
```

Comment:
CC @tonio I'd like to have a tooltip on this field, to explain a bit more. What's your opinion on https://www.w3schools.com/howto/howto_css_tooltip.asp ?

Reply:
There's already bootstrap, so I'd advise to use their tooltip system: https://getbootstrap.com/docs/3.4/javascript/#tooltips

Quickly grepin' the code, looks like it may already be used [here](https://github.com/georchestra/georchestra/blob/eab11000ea82717a5ce0ef8a5ea8221726e51fcf/console/src/main/webapp/manager/app/components/logger/logger.tpl.html#L92)

Reply:
Cool. Looks easy to implement !
@pmauduit would you like to give it a try ?

Reply:
> @pmauduit would you like to give it a try ?

![Screenshot from 2020-09-17 13-02-33](https://user-images.githubusercontent.com/594335/93462421-98538b80-f8e6-11ea-8ef3-a0fb5e04aa86.png)